frontend/src/components/Homepage/index.jsx

```diff
@@ -0,0 +1,34 @@
+import { Component } from 'react';
+import logo from '../../FrenchFryFoodCritic.png';
+import '../../App.css';
+
+class Homepage extends Component {
```

Comment:
We should look into whether or not we want to use functional or class components and probably change either this file or `frontend/src/components/Reviews/index.jsx` to match

[Article arguing for functional components](https://www.twilio.com/blog/react-choose-functional-components)